### PR TITLE
PLA-29176 - Fix remediation jobs for port rules

### PR DESCRIPTION
**Pull request by kshrutik**
Fix remediation jobs for port rules to remediate findings on a range of ports.

### CODE REVIEW

File: remediation_worker/jobs/ec2_close_port_1433/ec2_close_port_1433.py [hunk 1]
```diff
@@ -56,8 +56,207 @@ def parse(self, payload):
...
+                                    },
+                                ],
+                                'ToPort': rule["ToPort"]
+                            },
+                        ]
+                    )
+
     def remediate(self, client, instance_id):
-        """Block public access to port 1433 of all security groups attached to an EC2 instance.
+        """Block public access to port 22 of all security groups attached to an EC2 instance.
```
Comment (lytran2000) [inline remediation_worker/jobs/ec2_close_port_1433/ec2_close_port_1433.py]: Is this a cut & paste error as the file is ec2_close_port_1433, but comment mentions port 22 thru out the diff?

File: remediation_worker/jobs/ec2_close_port_1521/ec2_close_port_1521.py [hunk 2]
```diff
@@ -56,8 +56,207 @@ def parse(self, payload):
...
+                                    },
+                                ],
+                                'ToPort': rule["ToPort"]
+                            },
+                        ]
+                    )
+
     def remediate(self, client, instance_id):
-        """Block public access to port 1521 of all security groups attached to an EC2 instance.
+        """Block public access to port 22 of all security groups attached to an EC2 instance.
```
Comment (lytran2000) [inline remediation_worker/jobs/ec2_close_port_1521/ec2_close_port_1521.py]: This file tries to remediat port 1521 but comment mentions port 22.  May you check all files below?

Comment (kshrutik) [reply remediation_worker/jobs/ec2_close_port_1521/ec2_close_port_1521.py]: Sure! Thanks for catching it.

**Review by lytran2000 — APPROVED** [review]
Looking good.

**Review by nandeshguru — APPROVED** [review]

**Pull request merged**